frontend/src/components/scheduleEntry/dateHelperLocal.js

```diff
@@ -1,26 +1,20 @@
-function defineHelpers (scheduleEntry, timed = false) {
+function defineHelpers (dayjs, scheduleEntry, timed = false) {
```

Comment:
Any ideas how to make the `Vue.dayjs` function available in these helpers without passing it as a parameter? I read about mixins, but couldn't figure out how to use them for that case.

Reply:
Two Ideas without mixins:
- static access to the dayjs object, just with Vue.dayjs

Reply:
Thanks for pointing this out, I didn't know you get the augmented object by importing `vue`